bin/entry.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh
 if [ -z "${AWS_LAMBDA_RUNTIME_API}" ]; then
+    echo "ENTRY.SH: Running locally"
     exec /usr/bin/aws-lambda-rie $(which python) -m awslambdaric $1
 else
-    . /sync_lambda_envs.sh # Retrieve .env from parameter store and remove currently set environement variables
+    echo "ENTRY.SH: Running in AWS Lambda"
+    . /sync_lambda_envs.sh
+    env  # Optional: dump all environment variables
```

Comment:
This one might be dangerous, depending on where we run this and leak secrets.

Reply:
This thought occured to me as well -- but the reason for this was to actually show us in the logs what variables exist!  as it stands we have no idea what variables are exposed to the container because none of them show up in the lambda config env vars.

This gets run in the container entry script, and would show up in the lambda logs.  

I'm good to remove if you think we should, but i think it would also be helpful to have something to search for in cloudwatch when we want to verify our env vars :)